common/djangoapps/util/views.py

```diff
@@ -84,6 +84,7 @@ def inner(request, *args, **kwargs):
                     # In debug mode let django process the 500 errors and display debug info for the developer
                     raise
                 else:
+                    log.exception("Error in django view.")
```

Comment:
Is it possible to extract the exception message from the Exception object
and include that?
On Oct 22, 2015 2:05 AM, "Zia Fazal" <notifications@github.com> wrote:

> In common/djangoapps/util/views.py
> <https://github.com/edx/edx-platform/pull/10264#discussion_r42713952>:
>
> > @@ -84,6 +84,7 @@ def inner(request, *args, **kwargs):
> >                      # In debug mode let django process the 500 errors and display debug info for the developer
> >                      raise
> >                  else:
> > +                    log.exception("Error in django view.")
>
> @mattdrayer <https://github.com/mattdrayer> what should be the error
> message keeping in mind it is going to be a generic decorator for django
> views?.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/10264/files#r42713952>.
>